docs/modules/ROOT/pages/getting-started/connecting-to-neo4j.adoc

```diff
@@ -0,0 +1,119 @@
+[[connecting-to-neo4j]]
+:description: This tutorial demonstrates how to connect to Neo4j and utilize Cypher Builder.
```

Comment:
@lidiazuin domain, but to me, from the sentence seems the two objectives of the sentence are disjoined "Connect to Neo4j" and "Use Cypher Builder". I would go more with something like:
"This tutorial demonstrates how to execute your Cypher Builder queries in a Neo4j instance using the Neo4j Javascript driver".

Reply:
I guess we can go for something simpler, if correct:

```suggestion
:description: This tutorial demonstrates how to use the Cypher Builder to connect to Neo4j.
```

Reply:
In this way, it will make the sentence incorrect as you don't use Cypher Builder to connect to Neo4j, the component that "connects" is the Neo4j Javascript driver.

Reply:
aha, so your suggestion makes more sense!

Reply:
```suggestion
:description: "This tutorial demonstrates how to execute your Cypher Builder queries in a Neo4j instance using the Neo4j Javascript driver.
```

Reply:
Without the quotation marks in the beginning :)